spec/models/manageiq/providers/vmware/cloud_manager/vm_spec.rb

```diff
@@ -50,4 +50,33 @@
       vm.raw_destroy
     end
   end
+
+  describe 'power operations' do
+    before(:each) do
+      allow(ems).to receive(:with_provider_connection).and_yield(connection)
+    end
+
+    let(:ems) { FactoryGirl.create(:ems_vmware_cloud) }
+    let(:vm) { FactoryGirl.create(:vm_vcloud, :ext_management_system => ems, :ems_ref => 'id') }
+    let(:connection) { double('connection') }
+    let(:response) { double('response', :body => nil) }
```

Comment:
Nitpicking, but can you please vertically align `{`. I know that Rubocop says nothing, but still it's prettier.